eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SummaryStatistics.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * A Summarizer can be used to aggregate statistics for multiple primitive attributes.
+ *
+ * @since 8.1
  */
 public class SummaryStatistics<T> implements Procedure<T>
```

Comment:
Couple of comments for this class (and a comment for other 2 summary statistics classes, too)

- The statement `import org.eclipse.collections.impl.tuple.Tuples;` is not in use. 
- This class should have `serialVersionUID`
- Ideally we should have serialization tests for all 3 SummaryStatistics classes.